### ISSUE TITLE
All System Down

### ISSUE THREAD

**Issue by roseslove4all** [apilayer/positionstack#12]
Hi,
it's down from 2 days. still not fix.please fix it.

**Comment by TobbeG** [comment]
Still down 1.8.2023
Why is this so?

are paid plans down too?

**Comment by mtmail** [comment]
Yes, and it's not the first time they have a multiday outage. In the past even 12 full days (https://github.com/apilayer/positionstack/issues/6#issuecomment-1033054718)

**Comment by TobbeG** [comment]
anyone know if also down on paid plans?
